packages/meditrak-server/src/routes/exportSurveyResponses.js

```diff
@@ -257,13 +257,18 @@ export async function exportSurveyResponses(req, res) {
           questions.push(...extraQuestions);
         }
 
-        // Add the answers to be exported
+        // Add the questions info and answers to be exported
+        let exportRow = INFO_ROW_HEADERS.length + 1; // Add one to make up for header row
         for (let questionIndex = 0; questionIndex < questions.length; questionIndex++) {
           // Set up the left columns with info about the questions
           const question = questions[questionIndex];
-          const exportRow = INFO_ROW_HEADERS.length + questionIndex + 1; // Add one to make up for header row
           const questionInfo = infoColumnKeys.map(columnKey => question[columnKey]);
+          const questionType = questionInfo[1];
+          // Exclude 'SubmissionDate' and 'PrimaryEntity' rows from survey response export since these have no answers
+          if (NON_DATA_ELEMENT_ANSWER_TYPES.includes(questionType)) continue;
```

Comment:
Nice, I like using the existing relevant NON_DATA_ELEMENT_ANSWER_TYPES! One problem with this approach is that the card is only to exclude those 2 specific question types, so if we use `NON_DATA_ELEMENT_ANSWER_TYPES` we will also exclude the `INSTRUCTION` type and any future types that don't have data associated with them.

This is a good point though, do we want to exclude all the non-data-element types or just the two specified in the card. I think it is worth making a comment about on the card (tagging Edwin).

My personal feeling is that future `NON_DATA_ELEMENT_ANSWER_TYPES` should be excluded, but the `INSTRUCTION` type should not be excluded because instruction questions can be useful in reading a survey response.

Reply:
Therefore the line could read:
```suggestion
          if (NON_DATA_ELEMENT_ANSWER_TYPES.includes(questionType) && 
          questionType !== ANSWER_TYPES.INSTRUCTION) continue;
```